### ISSUE TITLE
Installation af OIOUBL i en North America database

### ISSUE THREAD

**Issue by CorpitalFDP** [MicrosoftDocs/dynamics365smb-docs#166]
Jeg har installeret NA-version af Business Central til en kunde  med kontor i både DK og USA og vil gerne installere OIOUBL, men den kan ikke installeres da upgrade codeunit forventer at det er en dansk database. 
Kan jeg få en version af OIOUBL.app der ikke indeholder codeunit 13623 "OIOUBL-MigrateToExtV2"

---
#### Dokumentoplysninger

⚠ *Undlad at redigere denne sektion. Det er påkrævet til docs.microsoft.com ➟ GitHub-problemlinkning.*

* ID: ae54faea-be1d-6d67-8b53-014504925f04
* Version Independent ID: 9a9d3538-a636-7c90-d53a-4a128470ed4a
* Content: [Oversigt over elektronisk OIOUBL-fakturering - Business Central](https://docs.microsoft.com/da-dk/dynamics365/business-central/localfunctionality/denmark/oioubl-electronic-invoicing-overview#feedback)
* Content Source: [business-central/LocalFunctionality/Denmark/oioubl-electronic-invoicing-overview.md](https://github.com/MicrosoftDocs/dynamics365smb-docs/blob/live/business-central/LocalFunctionality/Denmark/oioubl-electronic-invoicing-overview.md)
* Service: **dynamics365-business-central**
* GitHub Login: @SorenGP
* Microsoft Alias: **sgroespe**

**Comment by edupont04** [comment]
Hej CorpitalFDP,

I'm posting my reply in English so that others can benefit from it as well. The OIOUBL extension was created specifically for the Danish market. The current implementation means that it cannot be applied to a Business Central tenant in North America due to the way that Business Central handles local functionality. If you want to request a change to the extension, then you can suggest that here: https://github.com/microsoft/alappextensions/

**Comment by CorpitalFDP** [comment]
Hi Eva

What i your sugestion for a customer operating in several countrries and need localizations for more than one country. I have expected the benefit of moving localizations to apps was that we where able to pick the needed functionality to fit our customers needs.

**Comment by edupont04** [comment]
Hej igen,

Currently, companies operating in multiple countries are expected to sign up for Business Central in each country. As for the strategy and architecture behind moving local functionality to extensions, I must refer to the ALappextensions repo in GitHub that I linked to above. You are also very much welcome to suggest a change as an idea here: [https://experience.dynamics.com/ideas/categories/?forum=e288ef32-82ed-e611-8101-5065f38b21f1&forumName=Dynamics%20365%20Business%20Central](https://experience.dynamics.com/ideas/categories/?forum=e288ef32-82ed-e611-8101-5065f38b21f1&forumName=Dynamics%20365%20Business%20Central) 

Closing the issue for now.
